membership.lua

```diff
@@ -626,6 +626,9 @@ end
 local function leave()
     -- First, we need to stop all fibers
     local sock = _sock
+    if sock == nil then
+        return false
```

Comment:
Why `false`? The job is done. It should be `true`.

Reply:
You got `true` after first call. I suppose that if you consciously try to call "leave" twice you should get an error - `false`.